apps/paper-example/android/app/build.gradle

```diff
@@ -45,7 +45,7 @@ react {
 
     /* Hermes Commands */
     //   The hermes compiler command to run. By default it is 'hermesc'
-    // hermesCommand = "$rootDir/my-custom-hermesc/bin/hermesc"
+    hermesCommand = "../../node_modules/react-native/sdks/hermesc/osx-bin/hermesc"
```

Comment:
Why is it needed? From my experiences it's only necessary when you are doing some "custom" operations with Hermes.

Also, hardcoding `osx-bin` isn't a good idea if some of our contributors wanted to build it on Linux/Windows. AFAIK there was some way to put here a platform-agnostic path.

Reply:
+1